### ISSUE TITLE
Substrate Tasks

### ISSUE THREAD

**Issue by jtieri** [cosmos/relayer#975]
Once we have a working test case that demonstrates the creation of IBC light clients between a Substrate<>Cosmos chain, we can continue to iterate merging relevant PRs needed to drive the test case further (e.g. create connections, create channels, successful token transfers). When the entirety of the code relevant to Substrate support is merged into the feat branch we can begin to focus on merging into main and also add Cosmos<>Substrate conformance tests to the CI jobs to check for regressions.

- [ ] Merge PRs into feat branch required for BEEFY light client creation
    - [x] #923 
    - [ ] #942 
    - [ ] #946
- [ ] Build out test case in `ibctest` for spinning up chains and creating clients 
(see: https://github.com/strangelove-ventures/ibctest/blob/main/chain/polkadot/polkadot_chain_test.go)
- [ ] Continue to open PRs for the rest of the Substrate IBC ops necessary for ics20 support
- [ ] Merge feat branch into main
- [ ] Setup Cosmos<>Substrate conformance testing via `ibctest` in our CI

Getting everything merged up to client creation will unblock us on getting a testing case ready in `ibctest`. At that point we should be able to ditch the bash scripts that are currently being used which will significantly reduce the current overhead required for testing.

**Comment by jtieri** [comment]
This issue is very stale by now. Closing this
